src/widgets/requirements/ui/requirements.module.scss

```diff
@@ -0,0 +1,54 @@
+.content {
+  display: flex;
+  flex-direction: column;
+  gap: 32px;
+
+  width: 100%;
+  max-width: $content-width;
+  padding: 40px 120px;
+
+  @include media-laptop {
+    padding: 40px 40px;
+  }
+
+  @include media-tablet {
+    padding: 40px 16px;
+  }
+}
+
+.title {
+  margin-block-start: unset;
+  margin-block-end: unset;
+  margin-inline-start: unset;
+  margin-inline-end: unset;
+
+  font-size: 28px;
+  font-weight: $font-weight-regular;
+  line-height: 36px;
+  color: $color-gray-600;
+
+  @include media-laptop {
+    font-size: 22px;
+  }
+}
+
+.title-margin {
+  margin-bottom: 16px;
+}
```

Comment:
Instead of adding a new class, perhaps it would be better to set the indentation via the parent's gap property?

Reply:
Yes, I agree. But when committing alphabetical formatting in a css class, the margin rule for title does not work :)
Maybe your hint with the Subtitle component will help me solve this

Reply:
Thanks, using Subtitle solved the problem.
Done.